lib/xcprofiler/execution.rb

```diff
@@ -58,5 +60,9 @@ def line
         nil
       end
     end
+
+    def record_id
+      Digest::MD5.new.update("#{filename}#{method_name}").to_s
```

Comment:
I think this hash is not good.
Because, this implementation doesn't consider the case when there are same methods in the same file.

How about make hash from Position struct?